components/datagrid/DataGrid.js

```diff
@@ -196,10 +198,12 @@ DataGrid.propTypes = {
   stickyFromLeft: PropTypes.number,
   stickyFromRight: PropTypes.number,
   onSelectionChange: PropTypes.func,
+  processing: PropTypes.bool,
```

Comment:
A bit weird to call this prop `processing`, but it's adding a `LoadingBar`. Any reason to not call it `loading`?

Reply:
The `Toast` and `Button` both also use `processing` to render their `LoadingSpinner`s.

You could think about it as: the "`DataGrid` is processing your request".

In the end more then only rendering a `LoadingBar` should be happen if `processing` is true (e.g.: like diming the opacity of the body cells).
Although this case `loading` would also still be relevant.

Reply:
Alright, fair enough 👍